dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryHelper.java

```diff
@@ -54,4 +63,100 @@ public CategoryView toCategoryView(final Category category, final User user) {
                 .categoryVelocityVarName(category.getCategoryVelocityVarName())
                 .build();
     }
+
+    public void AddOrUpdateCategory(final User user, final String contextInode,final BufferedReader br,final Boolean merge) throws IOException, Exception {
+        CsvReader csvreader = new CsvReader(br);
```

Comment:
same thing all closables use try(CsvReader csvreader = new CsvReader(br);) {}

Reply:
this has been addressed - kindly have a look at the recent commit - thanks